apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -267,7 +289,7 @@ public function clear_sideloaded_image_cache( $id ) {
 	 * Deletes cached templates data when theme switches.
 	 */
 	public function clear_templates_cache() {
-		delete_transient( $this->get_templates_cache_key() );
+		delete_transient( $this->get_templates_cache_key( $this->get_verticals_locale() ) );
```

Comment:
Should we also clear template data that has been cached for other locales?

Reply:
The locale was always part of the cache key, so the previous behavior was only to delete the selected locale's transient. 

I didn't alter the default behavior as I'm not sure what the intention is here.